app/src/main/java/net/gsantner/markor/format/TextFormat.java

```diff
@@ -87,34 +92,38 @@ public static TextFormat getFormat(int formatId, Activity activity, Document doc
                 format.setConverter(CONVERTER_PLAINTEXT);
                 format.setHighlighter(new PlaintextHighlighter(hlEditor, document));
                 format.setTextActions(new PlaintextTextActions(activity, document));
+                format.setAutoFormatInputFilter(new MarkdownAutoFormat()); // Using the markdown syntax for plain text
+                format.setAutoFormatTextWatcher(new ListHandler(MarkdownAutoFormat.getPrefixPatterns()));
                 break;
             }
             case FORMAT_TODOTXT: {
                 format.setConverter(CONVERTER_TODOTXT);
                 format.setHighlighter(new TodoTxtHighlighter(hlEditor, document));
                 format.setTextActions(new TodoTxtTextActions(activity, document));
+                format.setAutoFormatInputFilter(new TodoTxtAutoFormat());
                 break;
             }
-
             case FORMAT_KEYVALUE: {
                 format.setConverter(CONVERTER_KEYVALUE);
                 format.setHighlighter(new KeyValueHighlighter(hlEditor, document));
                 format.setTextActions(new PlaintextTextActions(activity, document));
```

Comment:
if nothing new is set, like here - the old autoformat is kept. thus if you had todotxt before and to K-V, it uses todo.txt autoformat there.   
correct?